spec-tests/src/test/java/org/candlepin/spec/jobs/JobStatusSpecTest.java

```diff
@@ -0,0 +1,498 @@
+import org.candlepin.resource.OwnerApi;
+import org.candlepin.resource.OwnerProductApi;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.JobsClient;
+import org.candlepin.spec.bootstrap.client.SpecTest;
+import org.candlepin.spec.bootstrap.client.SpecTestFixture;
+import org.candlepin.spec.bootstrap.data.builder.Consumers;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.util.StringUtil;
+import org.candlepin.spec.bootstrap.data.util.UserUtil;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+/**
+ * Test the /status resource
+ */
+@SpecTest
+class JobStatusSpecTest extends SpecTestFixture {
+
+    private static ApiClient client;
+    ApiClient userClient;
+    private static OwnerDTO owner;
+    private static UserDTO user;
+    private OwnerApi ownerApi;
+    private OwnerProductApi ownerProductApi;
+    private ConsumerApi consumerApi;
+    private JobsClient jobsClient;
+    private ProductDTO product;
+    private PoolDTO pool;
+
+
+    @BeforeEach
+    public void beforeEach() throws ApiException {
+        client = ApiClients.admin();
+        ownerApi = client.owners();
+        ownerProductApi = client.ownerProducts();
+        jobsClient = getClient(JobsClient.class);
+        consumerApi = client.consumers();
+        owner = ownerApi.createOwner(Owners.random());
+        user = UserUtil.createUser(client, owner);
+        userClient = ApiClients.trustedUser(user.getUsername());
+
+
+        product = new ProductDTO();
+        product.setId(StringUtil.random("ID"));
+        product.setName(StringUtil.random("Test Product"));
+        product = ownerProductApi.createProductByOwner(owner.getKey(), product);
+
+        pool = new PoolDTO();
+        pool.setProductId(product.getId());
+        pool.setSubscriptionId(StringUtil.random("source_sub"));
+        pool.setSubscriptionSubKey(StringUtil.random("sub_key"));
+        pool.setUpstreamPoolId(StringUtil.random("upstream"));
+        pool.setQuantity(4L);
+        pool.setStartDate(Instant.now().atOffset(ZoneOffset.UTC));
+        pool.setEndDate(Instant.now().plus(365, ChronoUnit.DAYS).atOffset(ZoneOffset.UTC));
+        pool = ownerApi.createPool(owner.getKey(), pool);
+    }
+
+    @Test
+    @DisplayName("should find an empty list if the owner key is wrong")
+    public void shouldFindEmptyListOwnerNotExist() throws Exception {
+        List<AsyncJobStatusDTO> jobs = jobsClient.listMatchingJobStatusForOrg("totally-made-up",
+            null, null);
+        assertThat(jobs == null || jobs.size() == 0);
```

Comment:
AssertJ's `assertThat` works a bit differenty. You put what you want to test into `assertThat` and follow with a chain of contextual assertions.

Some examples:
```java
assertThat(jobs).isEmpty();
assertThat(jobs)
    .isNotNull()
    .isNotEmpty();
assertThat(jobs).hasSize(10);
assertThat(id).isEqualTo(anotherId);
```